spec/controlbox.js

```diff
@@ -1087,6 +1087,33 @@
             // XXX: Awaiting more tests, close it again for now...
             panel.$el.find('a.toggle-xmpp-contact-form').click();
         }));
+
+        it("can be added to roster and check for case-sensivity", mock.initConverse(function (_converse) {
```

Comment:
Oh! I didn't know that. Should I write something like
""The 'Add Contact widget" can be used to add contacts to roster and it check for case-sensitivity"
?